### PR TITLE
Memoize ReanimatedModule in `sendEvent`

### PR DESCRIPTION
**Pull request by tomekzaw**
## Summary

This PR adds `static __weak REAModule *reaModule;` variable which stores the result of `[[self bridge] moduleForName:@"ReanimatedModule"]` so we can call it only once during initialization instead of on each event.

When `REAModule` is deallocated, all `__weak` pointers are automatically replaced with `nil` so it doesn't leak any resources as well as works fine after a reload.

### CODE REVIEW

File: ios/REAEventDispatcher.m [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 #import <React/RCTBridge+Private.h>
 #import <React/RCTDefines.h>
 
 @implementation REAEventDispatcher
 
 - (void)sendEvent:(id<RCTEvent>)event
 {
-  [[[self bridge] moduleForName:@"ReanimatedModule"] eventDispatcherWillDispatchEvent:event];
+  __weak static REAModule *reaModule;
+  __weak static RCTBridge *lastBridge;
```
Comment (piaskowyk) [inline ios/REAEventDispatcher.m:13]: What about retain cycle? Can we keep refference to bridge?

Comment (piaskowyk) [reply ios/REAEventDispatcher.m:13]: You are right I missed it. 👍 

Just to be sure - did you tested those changes with continues events (for example scrolling) and app reload?

Comment (tomekzaw) [reply ios/REAEventDispatcher.m:13]: Nope, not yet, but I will, thanks! Honestly, I'm a bit afraid about this change, not sure if the performance gain is worth it.

Comment (tomekzaw) [reply ios/REAEventDispatcher.m]: Reloading while scrolling works fine, tested on ScrollViewExample.tsx.

**Pull request merged**